<!--
Coordinate Transform
X:  Forward
Y:  Left
Z:  Up
-->

<launch>
    <param name="robot_description" textfile="/home/automation/sick_scan_ws/src/bsl_pkg/urdf/CTA.urdf" />
    
    <!--///Transforms///-->
    <include file="$(dirname)/transforms.launch" />
    
    <!--///Droper///-->
    <node pkg="topic_tools" type="drop" name="drop" args="cloud 3 4"/>
    
    <!--///Bag Playback/// forklifttest1_notf or taptest-->
    <node pkg="rosbag" type="play" name="play" args="-l -q /home/automation/forklifttest1_notf.bag"/>
    
    <node pkg="rviz" type="rviz" name="rviz"/>
    
    <node pkg="bsl_pkg" type="tf2_laser_angle_calibrator.py" name="tf2_laser_angle_calibrator" output="screen" launch-prefix="xterm -e"/>
</launch>
